src/extensions/default/CodeFolding/main.less

```diff
@@ -63,11 +57,11 @@
     border: 1px solid @color-marker-border;
     color: @color-marker-text;
     background-color: @color-marker-bg;
-    
+
     &:after {
         content: "...";
         font-size: 1em;
         vertical-align: top;
         line-height: 0.8em;
     }
-}
```

Comment:
let me know if I have to rollback everything and send it again, I got confused because I was using hg to send some patch to firefox, the workflows are similar yet different so I got confused. sorry. 😢